datasets/ethos/README.md

```diff
@@ -0,0 +1,16 @@
+# ETHOS Hate Speech Dataset
```

Comment:
can you add the yaml tags at the top file the dataset card please ?
You can find more infos here https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card

Also can you follow the template for the dataset card ? you can leave some sections content with `[More Information Needed]`: https://github.com/huggingface/datasets/blob/master/templates/README.md

Reply:
@lhoestq Hello! I added the PR1453 addressing all the issues you mentioned 😊! Thanks